src/themes/basic/_coverpage.styl

```diff
@@ -15,8 +15,8 @@ section.cover
     opacity 0.8
     position absolute
     top 0
-    height 100%
-    width 100%
+    height 100vh
+    width 100vw
```

Comment:
If we do this, due to how `vw` works, we need to verify that the page does not start scrolling sideways, because `100vw` does not account for scrollbar width. See this:

https://stackoverflow.com/questions/23367345/100vw-causing-horizontal-overflow-but-only-if-more-than-one/23367686

Can you add screenshots in this PR?

Reply:
I added screenshots.